django/cantusdb_project/main_app/views/views.py

```diff
@@ -576,4 +585,42 @@ def change_password(request):
         form = PasswordChangeForm(request.user)
     return render(request, 'registration/change_password.html', {
         'form': form
-    })
\ No newline at end of file
+    })
+
+def pm_check(user):
```

Comment:
Can you add some comments/doc strings indicating the necessity of this function where this function is being used?